### PR TITLE
More testing

### PR DESCRIPTION
**Pull request by Shane32**
This PR adds end-to-end testing for SampleWebApp so that the Program and Startup files are also tested.

This currently targets the `tlil-fix-uniqueness-of-constructed-graph-types` branch.  If merged, it will integrate into #242 .  Or if #242 is merged first, then this will target `master`.

### CODE REVIEW

File: samples/SimpleWebApp.Tests/EndToEndTests.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
+using System.Net;
+using System.Text.Json;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+
+namespace SubscriptionExample.Tests;
```
Comment (tlil) [inline samples/SimpleWebApp.Tests/EndToEndTests.cs]:
```suggestion
namespace SimpleWebApp.Tests;
```

**Review by tlil — APPROVED** [review]

**Comment by Shane32** [comment]
@tlil Feel free to merge this into #242 now, or merge #242 first and I'll fix up this PR to separately merge into master.

**Comment by tlil** [comment]
Thanks. Merging into mine now.

**Comment by sawakatansari1-cmd** [comment]
Vkysufkvysgjvudydhcnchfhcjc rnc

**Comment by sawakatansari1-cmd** [comment]
Jpftivm gsgkxkhsrusgxb nvlhuxj m yfkv

**Pull request merged**